migration/iD10/oracle/202203160709_IDEMPIERE-5232.sql

```diff
@@ -0,0 +1,18 @@
+-- IDEMPIERE-5232
+SELECT register_migration_script('202203160709_IDEMPIERE-5232.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Mar 16, 2022, 7:09:20 AM CET
+UPDATE AD_Table SET AccessLevel='6',Updated=TO_TIMESTAMP('2022-03-16 07:09:20','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Table_ID=53284
```

Comment:
When changing AccessLevel it is required to generate the I_ class because the variable accessLevel is hardcoded.

That's something I think would be good to improve - to read the access level from database instead of this hardcoded variable - @hengsin  - WDYT ?

Reply:
hi @CarlosRuiz-globalqss , I think we can try changing Model and accessLevel to static final and get accessLevel from db.